src/lib/slider/slider.ts

```diff
@@ -283,13 +360,20 @@ export class MdSlider implements ControlValueAccessor {
         this.value = this.min;
         break;
       case LEFT_ARROW:
-        this._increment(-1);
+        // It's kind of debatable what's the correct thing to do for inverted sliders. For a sighted
```

Comment:
I'd change "kind of debatable" bit and simply say that, for inverted sliders, we care more about the a11y experience than making having the keyboard shortcuts "look right" for sighted users. (I do think this is the right call).